buildSrc/DevBuild.js

```diff
@@ -23,12 +23,14 @@ export async function runDevBuild({ stage, host, desktop, clean, ignoreMigration
 	const isCalendarBuild = app === "calendar"
 	const tsConfig = isCalendarBuild ? "tsconfig-calendar-app.json" : "tsconfig.json"
 	const buildDir = isCalendarBuild ? "build-calendar-app" : "build"
+	const liboqsIncludeDir = "libs/webassembly/include"
 
 	console.log("Building dev for", app)
 
 	if (clean) {
 		await runStep("Clean", async () => {
 			await fs.emptyDir(buildDir)
+			await fs.rm(liboqsIncludeDir, { recursive: true, force: true })
```

Comment:
this will cause `make` to bulid every time if I understand it right

Reply:
Yes, in case of calling `node make --clean" not without the `clean`flag. We made this change and https://github.com/tutao/tutanota/pull/8440/commits/e3930840970fd82641db3b911a13703eae5c66a3#diff-266f063a62e86cb7039f2c34e74c6a3f80dfe2f91202a909cebb2b1671cc9c30R34 to ensure that devs do not have to manually delete folders on their machines or update submodules manually. Any concerns?

Reply:
I didn't see that this was part of the clean, sorry